File: chap11/fig11-3.R
d <- read.csv(file='input/data-weight.csv')

png('output/fig11-3.png', pointsize=48, width=1200, height=900)
par(mar=c(4, 4, 0.1, 0.1))
acf(d$Y, lwd=15, lend=1, main='')
dev.off()

File: chap11/fig11-3.py
import pandas
import matplotlib.pyplot as plt
import statsmodels.api as sm

plt.style.use('ggplot')

d = pandas.read_csv('input/data-weight.csv')
fig = plt.figure(figsize=(6, 4))
ax1 = fig.add_subplot(111)
sm.graphics.tsa.plot_acf(d.Y, lags=13, ax=ax1)
plt.savefig('output/fig11-3.py.png', dpi=300)
